stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/core.py

```diff
@@ -89,7 +88,7 @@ def item_collection(
                 .order_by(self.item_table.datetime.desc(), self.item_table.id)
             )
             count = None
-            if self.extension_is_enabled(ContextExtension):
+            if self.extension_is_enabled("ContextExtension"):
```

Comment:
What's the reasoning behind this change?

Reply:
This change links somewhat to the thinking which spawned #178. It would be really good to try and un-bake the extensions from the core client. The reasoning behind this change is to remove the need to import the Extension classes to check if the extension is active. The client instance has the Extension instances passed to it at runtime so I changed the `is_extension_enabled` method to enable simple name checking https://github.com/rsmith013/stac-fastapi/blob/247098fbb28345096080ea161158ce4461868abb/stac_fastapi/types/stac_fastapi/types/core.py#L127